app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -233,6 +242,30 @@ private void showFeed() {
         }
 
         txtvUrl.setText(feed.getDownload_url() + " {fa-paperclip}");
+
+        if (feed.getPaymentLinks() == null || feed.getPaymentLinks().size() == 0) {
+            lblSupport.setVisibility(View.GONE);
+            txtvFundingUrl.setVisibility(View.GONE);
+        } else {
+            lblSupport.setVisibility(View.VISIBLE);
+            ArrayList<FeedFunding> fundingList = feed.getPaymentLinks();
+            String str = "";
+            ArrayList<String> seen = new ArrayList<String>();
+            for (FeedFunding funding : fundingList) {
+                if (seen.indexOf(funding.url) >= 0) {
+                    continue;
+                }
+                seen.add(funding.url);
+                str += (funding.content.isEmpty()
```

Comment:
Without having looked at this in Android Studio, I think it will display a warning and suggest to use StringBuilder instead, so that less data needs to be copied.

Reply:
that is correct.